### PR TITLE
Task 3 : Design a Data collection tool to record symptoms Data

### PR DESCRIPTION
**Pull request by shivani6320**
- [x] Outreachy Applicant

- Created a folder under name [Shivani's Contribution] in Task 3's Applicant's Contributions folder.
-  designed data collection tool to record the symptoms data.
- added README file for brief overview.

### CODE REVIEW

**Comment by shivani6320** [comment]
Hey @Mboalab ! Can you please review my pull request and give your valuable feedback?

File: Starter Tasks/Newcomer Tasks/Task 3: Design a Data collection tool to record Typhoid Fever symptoms Data/Applicants Folder/Shivani's Contributions/README.md [hunk 1]
```diff
@@ -1 +1,9 @@
-# TASK 3
+# TASK 3 :  Data Collection Tool
+
+- The selected tool for collecting reliable fitting data is 'Google Forms'
+- I have created a questionnaire which takes answers in form of numerical as well as categorical data.
+- The form can be accessed from [here](questionnaire)
```
Comment (Mboalab) [inline Starter Tasks/Newcomer Tasks/Task 3: Design a Data collection tool to record Typhoid Fever symptoms Data/Applicants Folder/Shivani's Contributions/README.md:5]: the links to the questionnaire seems not working...please can you fix this?

File: Starter Tasks/Newcomer Tasks/Task 3: Design a Data collection tool to record Typhoid Fever symptoms Data/Applicants Folder/Shivani's Contributions/README.md [hunk 2]
```diff
@@ -1 +1,9 @@
-# TASK 3
+# TASK 3 :  Data Collection Tool
+
+- The selected tool for collecting reliable fitting data is 'Google Forms'
```
Comment (Mboalab) [inline Starter Tasks/Newcomer Tasks/Task 3: Design a Data collection tool to record Typhoid Fever symptoms Data/Applicants Folder/Shivani's Contributions/README.md:3]: Google form is great...but do you have any idea for an offline solution. Specifically for Physicians working in rural zone without electricity or internet connexion.

File: Starter Tasks/Newcomer Tasks/Task 3: Design a Data collection tool to record Typhoid Fever symptoms Data/Applicants Folder/Shivani's Contributions/README.md [hunk 3]
```diff
@@ -1 +1,9 @@
...
+- I have created a questionnaire which takes answers in form of numerical as well as categorical data.
```
Comment (Mboalab) [inline Starter Tasks/Newcomer Tasks/Task 3: Design a Data collection tool to record Typhoid Fever symptoms Data/Applicants Folder/Shivani's Contributions/README.md:4]: Good ...I advise you also to see how data collected from google form are aligned with "Structured Data"

Comment (shivani6320) [reply Starter Tasks/Newcomer Tasks/Task 3: Design a Data collection tool to record Typhoid Fever symptoms Data/Applicants Folder/Shivani's Contributions/README.md:4]: Sure! The collected data can be stored in structural rows and columns form in a spreadsheet format .

**Pull request closed without merging**